### PR TITLE
fix: tooltip left class for user menu

### PR DESCRIPTION
**Pull request by GusBaamonde**
tooltip-left-user-menu-01

tooltip-left-user-menu-03

tooltip-left-user-menu-02

#Checklist

- [X] Check if there is a new color.
- [X] Create the color variable in the _colors.scss file.
- [X] Follow the color nomenclature.
- [X] Create the color class with its variable.
- [X] Create the color component in the html, show the name, color class and its exadecimal.

### CODE REVIEW

File: assets/templates/header-component.html [hunk 1]
```diff
@@ -167,6 +167,27 @@
...
                         </button>
                       </div>
+                      <div class="user-plan--type">
+                        <p><strong>498</strong> Suscriptores disponibles</p>
+                        <div class="dp-request-sent">
+                          <button
+                            class="user-plan close-user--menu"
+                            type="button"
+                          >
+                            solicitud enviada
```
Comment (nreal) [inline assets/templates/header-component.html:177]: This text is only text or label, maybe we need to add css for <p> or <label>, see https://makingsense.atlassian.net/wiki/spaces/DOP/pages/1390051366/Flujo+de+Contrataci+n+de+Plan+Junio+2021

Comment (GusBaamonde) [reply assets/templates/header-component.html:177]: I did not understand the question.
I'll call you and we'll see it if you want. 👍

File: assets/templates/header-component.html [hunk 2]
```diff
@@ -167,6 +167,23 @@
                           comprar
                         </button>
                       </div>
+                      <div class="user-plan--type">
+                        <p><strong>498</strong> Suscriptores disponibles</p>
+                        <div class="dp-request-sent">
+                          <button
```
Comment (cbernat) [inline assets/templates/header-component.html:173]: If changes are added into the html take in accound that they should be added into webapp too. 
To validate what I'm asserting please navigate into new reports to check how this dropdown is showing.

Comment (GusBaamonde) [reply assets/templates/header-component.html:173]: just add a new item and we check it against previous versions. They are all in the library.
Being a new element, it should not impact what is already there.
I tried the library side in all the cases we use and it works fine
@nreal will have to test how it looks on the screens you mention.

- https://cdn.fromdoppler.com/doppler-ui-library/build1241/nav-user-component.html
- https://cdn.fromdoppler.com/doppler-ui-library/build1241/header-component.html#

**Pull request merged**